packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -82,10 +82,11 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => ({
   `,
 }));
 
-interface Props {
+export interface Props {
   isOpen?: boolean;
   label: string;
   loading?: boolean;
+  /** Toggle collapsed header icon */
   collapsible?: boolean;
   onToggle?: (isOpen: boolean) => void;
```

Comment:
Per the definition of done, all props should have a comment. See component request issue template